Doc/Tutorial/chapter_cluster.tex

```diff
@@ -370,20 +366,15 @@ \subsection*{$k$-medoids clustering}
 \begin{itemize}
 \item as a 2D Numerical Python array (in which only the left-lower part of the array will be accessed):
 \begin{minted}{python}
-distance = array([[0.0, 1.1, 2.3],
-                  [1.1, 0.0, 4.5],
-                  [2.3, 4.5, 0.0]])
+distance = array([[0.0, 1.1, 2.3], [1.1, 0.0, 4.5], [2.3, 4.5, 0.0]])
```

Comment:
This change (and the similar ones later in this file) is arguably less readable - all in one line for a matrix